Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm

```diff
@@ -171,6 +175,37 @@
 #endif // HAVE (INLINE_PREDICTIONS)
 }
 
+static RetainPtr<NSDictionary> unarchivedTokenForData(RetainPtr<NSData> tokenData)
+{
+    NSError *error = nil;
+    return [NSKeyedUnarchiver unarchivedObjectOfClasses:[NSSet setWithObjects:[NSDictionary class], [NSNumber class], [NSString class], nil] fromData:tokenData.get() error:&error];
+}
+
+std::span<const uint8_t> AXRemoteFrame::generateRemoteToken() const
+{
+    return span(Accessibility::newAccessibilityRemoteToken([[NSUUID UUID] UUIDString]));
+}
+
+void AXRemoteFrame::initializePlatformElementWithRemoteToken(std::span<const uint8_t> token, int processIdentifier)
+{
+    m_processIdentifier = processIdentifier;
+
+    RetainPtr nsToken = toNSData(token);
+    NSDictionary *tokenDictionary = nsToken ? unarchivedTokenForData(nsToken).get() : nullptr;
+    if (!tokenDictionary)
+        return;
+
+    NSString *uuid = [tokenDictionary objectForKey:@"ax-uuid"];
+    AXRemoteElement* remoteElement = [[AXRemoteElement alloc] initWithUUID:uuid andRemotePid:processIdentifier andContextId:0];
+    remoteElement.onClientSide = YES;
+    remoteElement.accessibilityContainer = parentObject() ?  parentObject()->wrapper() : nil;
```

Comment:
Should this be `parentObjectUnignored()`? That's what we use in `-[WebAccessibilityObjectWrapperIOS accessibilityContainer]`. But maybe this context is special?

Seems like there's an extra space after the `?`

Reply:
I used `parentObject()` here in case the scrollview/iframe was ignored, but that doesn't seem possible anymore. I will update this to use `parentObjectUnignored`.